lib/salus/scanners/gosec.rb

```diff
@@ -0,0 +1,50 @@
+require 'salus/scanners/base'
+# Gosec scanner check Go for insecure coding patters.
+# https://github.com/securego/gosec
+
+module Salus::Scanners
+  class Gosec < Base
+    def run
+      # Shell Instructions:
+      #   - -fmt=json for JSON output
+      #   - gosec only successfully scans repos within $GOPATH, we force the
+      #     go path reference via symlink within Dockerfile & Dockerfile.tests
+      if ENV['RUNNING_SALUS_TESTS'] == "true"
```

Comment:
Can we avoid this branching if we add `/home/repo` to GOPATH instead of symlinking it?

Reply:
From the digging and testing I did, it looks like we are stuck with this solution until gosec figures how to support go modules (aka until gosec stops requiring we run repo against a `$GOPATH`).  reference: https://github.com/securego/gosec/issues/234#issuecomment-441569937